### DISCUSSION TITLE
How to batch process multiple json files of labelme?

### DISCUSSION THREAD

**Discussion by jannatbedi** [wkentaro/labelme#1139] [Q&A / Help] [answered]
I have a folder which consists of the images as well as their corresponding json files generated via labelme. I want to generate the label.png of all these files with a single command instead of running a command for each individual json file. How do I do that?

**Comment by wkentaro** [comment, marked as answer]
Use labelme2voc.py in the examples folders.

**Comment by openXG** [comment]
There are many type of json_to_dataset.py to generate all your json files on CSDN. 
but I found that the color changed and  some can't matched...
@wkentaro By the way, could I change target parts color? I just watched the issue #216 but I still don't know how to fix this problem, I just need gray and white label.
Looking forward to your help and much thx!

**Comment by wkentaro** [comment]
what is target parts color?

**Comment by jannatbedi** [comment]
Thank you!

**Comment by openXG** [comment]
Oh sorry. my English is poor.
I mean that I get same problem with issue #216, and I left a comment under issue #217

**Comment by wkentaro** [comment]
@openXG OK. You can change `colormap` here:
https://github.com/wkentaro/labelme/blob/caa54a707966cdc62e4ac9d3569e17aa4e90104b/examples/semantic_segmentation/labelme2voc.py#L54
https://github.com/wkentaro/labelme/blob/caa54a707966cdc62e4ac9d3569e17aa4e90104b/examples/semantic_segmentation/labelme2voc.py#L84-L85

**Comment by openXG** [comment]
be deep grateful!

**Comment by wkentaro** [comment]
:+1:
